plotly/figure_factory/_violin.py

```diff
@@ -435,6 +440,8 @@ def create_violin(data, data_header=None, group_header=None, colors=None,
         number and will be used to color the violin plots if a colorscale
         is being used.
     :param (bool) rugplot: determines if a rugplot is draw on violin plot.
+    :param (bool) forced_order: determines if violins are sorted
+        alphabetically (True) or by inputted order (False).
```

Comment:
input order

@Kully can you add defaults to this doc string (like https://github.com/plotly/plotly.py/blob/master/plotly/figure_factory/_distplot.py#L63-L70). It seems a little confusing to list True before False though False is the default